test/runtests.jl

```diff
@@ -491,24 +529,24 @@ end
 
     ft = ftest(mod, nullmod)
     @test isapprox(ft.pval[1].v,  2.481215056713184e-8)
-    @test sprint(show, ftest(mod, nullmod)) ==
```

Comment:
I'm not sure how I feel about this. Editors which automatically remove trailing spaces are annoying anyway since they usually introduce lots of whitespace changes in unrelated areas. So maybe not worth making the code more complex to cope with them.

Reply:
Done! (I've undone the changes.)